database/models/datum.go

```diff
@@ -24,3 +24,15 @@ type Datum struct {
 func (Datum) TableName() string {
 	return "datum"
 }
+
+// PlutusData represents a Plutus data value in the witness set
+type PlutusData struct {
+	ID            uint   `gorm:"primaryKey"`
+	TransactionID uint   `gorm:"index"`
+	Data          []byte `gorm:"type:bytea"`
```

Comment:
Remove the GORM field type override. This is a Postgres column type that won't work in SQLite